joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/CallTests.scala

```diff
@@ -131,7 +134,47 @@ class NewCallTests extends JavaSrcCode2CpgFixture {
         |}
         |""".stripMargin)
 
-    cpg.call.name("foo").methodFullName.l shouldBe List("foo.Foo.foo:java.lang.String()")
+    inside(cpg.call.name("foo").l) { case List(fooCall) =>
+      fooCall.methodFullName shouldBe "foo.Foo.foo:java.lang.String()"
+      fooCall.staticReceiver shouldBe Some("foo.Foo")
+    }
+  }
+
+  "calls to inherited static methods" should {
```

Comment:
Please also add a test where the static method is called on an instance and not directly via the class name.